src/legacy/core_plugins/kibana/public/visualize/_index.scss

```diff
@@ -6,6 +6,6 @@
 //   visChart__legend-isLoading
 
 @import './editor/index';
-@import './embeddable/index';
+@import '../../../visualizations/public/embeddable/index';
```

Comment:
Instead of going so far out of this folder to import the moved sass files, can you just add a new `index.scss` file inside  [`/src/legacy/core_plugins/visualizations/public`](https://github.com/elastic/kibana/tree/c78cae688254044068003d3889fc215f737bb41e/src/legacy/core_plugins/visualizations/public) that imports the sass files within?

You can follow the same pattern as the [`/src/legacy/core_plugins/vis_type_table/public/`](https://github.com/elastic/kibana/tree/c78cae688254044068003d3889fc215f737bb41e/src/legacy/core_plugins/vis_type_table/public) folder.